### PR TITLE
a small script to add users to ldap groups

### PR DESCRIPTION
**Pull request by abizer**
doing `kinit abizer/admin ldapvi cn=ocfhpc` or whatever isn't hard, but this is slightly nicer

### CODE REVIEW

**Comment by abizer** [comment]
hmm, yeah. I should also check `klist`

**Comment by redplanks** [comment]
How does it fail if you supply an invalid group and/or user?

**Comment by dkess** [comment]
IMO we should have a confirmation dialog ("are you sure?") since this has the potential to do dangerous stuff.

Otherwise lgtm, we should perhaps also modify necessary parts of our docs to use this command.

**Comment by cg505** [comment]
@dkess++, maybe we should print the change before actually sending it to ldapmodify

File: staff/acct/add-to-group [hunk 1]
```diff
@@ -1,14 +1,33 @@
 #!/bin/bash
 set -euo pipefail
 
+if [ "$#" -ne 2 ]; then
+    ehco "usage: $0 username groupname"
```
Comment (dkess) [inline staff/acct/add-to-group]: typo

**Pull request merged**